notifications_api_common/constants.py

```diff
@@ -0,0 +1,12 @@
+# Exernally defined scopes.
+SCOPE_NOTIFICATIES_CONSUMEREN_LABEL = "notificaties.consumeren"
+SCOPE_NOTIFICATIES_PUBLICEREN_LABEL = "notificaties.publiceren"
+
+# Known channel names
+KANAAL_AUTORISATIES = "autorisaties"
+KANAAL_BESLUITEN = "besluiten"
+KANAAL_BESLUITTYPEN = "besluittypen"
+KANAAL_DOCUMENTEN = "documenten"
+KANAAL_INFORMATIEOBJECTTYPEN = "informatieobjecttypen"
+KANAAL_ZAAKTYPEN = "zaaktypen"
+KANAAL_ZAKEN = "zaken"
```

Comment:
why do we need these?

Reply:
Hmm good question, I think we don't :sweat_smile: